iam_builder/exceptions.py

```diff
@@ -0,0 +1,2 @@
+class IAMValidationError(Exception):
```

Comment:
Just wondering if we should be this inherit from the jsonschema ValidationError class? That way any usage of IAM builder where someone does a try except on a ValidationError will still be caught?

Reply:
That's a fair point, I did originally have it like that but changed it to make it simpler.